ambry-server/src/integration-test/java/com/github/ambry/server/ServerTestUtil.java

```diff
@@ -479,6 +484,72 @@ static void endToEndTest(Port targetPort, String routerDatacenter, MockCluster c
       releaseNettyBufUnderneathStream(stream);
       assertEquals(ServerErrorCode.Bad_Request, adminResponseWithContent.getError());
 
+      //
+      // Start the test of ForceDeleteAdminRequest
+      //
+      // Force delete on a blob which doesn't exist.
+      short lifeVersion = 1;
+      long nowTime = System.currentTimeMillis();
+      ForceDeleteAdminRequest forceDeleteAdminRequest = new ForceDeleteAdminRequest(blobId6, lifeVersion,
+          new AdminRequest(AdminRequestOrResponseType.ForceDelete, partitionIds.get(0), 1, "clientid2"));
+      stream = channel.sendAndReceive(forceDeleteAdminRequest).getInputStream();
+      AdminResponse adminResponse = AdminResponse.readFrom(stream);
+      releaseNettyBufUnderneathStream(stream);
+      assertEquals(ServerErrorCode.No_Error, adminResponse.getError());
+      // Use BlobIndexAdminRequest to verify we have one message info.
+      blobIndexAdminRequest = new BlobIndexAdminRequest(blobId6,
+          new AdminRequest(AdminRequestOrResponseType.BlobIndex, partitionIds.get(0), 1, "clientid2"));
+      stream = channel.sendAndReceive(blobIndexAdminRequest).getInputStream();
+      adminResponseWithContent = AdminResponseWithContent.readFrom(stream);
+      releaseNettyBufUnderneathStream(stream);
+      assertEquals(new String(adminResponseWithContent.getContent()), ServerErrorCode.No_Error,
+          adminResponseWithContent.getError());
+      jsonBytes = adminResponseWithContent.getContent();
+      objectMapper = new ObjectMapper();
```

Comment:
we don't have to create objectMapper again.

Reply:
It was due to some extra new ObjectManager as the other comment.